src/nidcpower/metadata/nidcpower.proto

```diff
@@ -46,7 +46,6 @@ service NiDCPower {
   rpc ConfigureLCRCustomCableCompensation(ConfigureLCRCustomCableCompensationRequest) returns (ConfigureLCRCustomCableCompensationResponse);
   rpc ConfigureOutputEnabled(ConfigureOutputEnabledRequest) returns (ConfigureOutputEnabledResponse);
   rpc ConfigureOutputFunction(ConfigureOutputFunctionRequest) returns (ConfigureOutputFunctionResponse);
-  rpc ConfigureOutputRange(ConfigureOutputRangeRequest) returns (ConfigureOutputRangeResponse);
```

Comment:
Grabbed the latest copy of the .proto from the grpc-device repo.
This entrypoint removal doesn't matter. The Python API doesn't support it, anyway.